server/utils/outOfServiceBedUtils.test.ts

```diff
@@ -193,4 +194,58 @@ describe('outOfServiceBedUtils', () => {
       ).toEqual(`${outOfServiceBedsForToday.length} beds`)
     })
   })
+
+  describe('bedRevisionDetails', () => {
+    it('adds a start date the summary list', () => {
+      const startDate = new Date(2024, 2, 1)
+      const revision = outOfServiceBedRevisionFactory.build({
+        startDate: DateFormats.dateObjToIsoDate(startDate),
+      })
+
+      expect(bedRevisionDetails(revision)).toEqual(
+        expect.arrayContaining([
+          { key: { text: 'Start date' }, value: { text: DateFormats.dateObjtoUIDate(startDate) } },
+        ]),
+      )
+    })
+
+    it('adds an end date the summary list', () => {
```

Comment:
is it worth making more explicit that the date is 'formatted' as well as 'added'? .e.g

`'adds a formatted end date to the summary list'`